client/src/app/routes/chrysalis/Message.tsx

```diff
@@ -173,7 +173,7 @@ class Message extends AsyncComponent<RouteComponentProps<MessageRouteProps>, Mes
                                                 this.state.paramMessageId
                                             )}
                                             buttonType="copy"
-                                            labelPosition="top"
+                                            labelPosition="right"
```

Comment:
We should make it "right" by default if no prop is passed, so we can remove the prop from all these places.

Reply:
Done.